src/store/useAppStore.test.ts

```
@@ -295,3 +295,65 @@ describe('useAppStore repository performance guards', () => {
+    useAppStore.getState().setGitHubToken('ghp_xxx');
+    useAppStore.getState().setUser(user);
+
+    const raw = window.localStorage.getItem(AUTH_MIRROR_KEY);
+    const mirror = JSON.parse(raw || '{}');
+    expect(mirror.githubToken).toBe('ghp_xxx');
+    expect(mirror.user.login).toBe('test-user');
+  });
+
+  it('persists backendApiSecret to the mirror and clears on logout', () => {
+    useAppStore.getState().setBackendApiSecret('secret-1');
+    let parsed = JSON.parse(window.localStorage.getItem(AUTH_MIRROR_KEY) || '{}');
```

ESLint

[error] 'parsed' is never reassigned. Use 'const' instead.

(prefer-const)